addons/udes_stock/models/stock_picking.py

```diff
@@ -820,6 +820,12 @@ def add_unexpected_parts(self, product_quantities):
         """
         raise ValidationError(_("Cannot handle over receiving!"))
 
+    def action_cancel(self):
+        """Extend to set date_done field When cancelling transfers"""
+        res = super().action_cancel()
+        self.write({'date_done': fields.Datetime.now(), 'priority': '0'})
```

Comment:
nit: Black format.

Reply:
Does this in core Odoo too, we'll probably need to review this.

In Odoo terms it sets it to normal priority, so something cancelled/done can never be urgent... can see where they are coming from but conflicts against the way we want to use priorities.

Reply:
Will check this again in the morning, but probably something to handle in another story, makes sense to make it consistent with completed pickings atm I think.

Reply:
Ah yeah - https://github.com/unipartdigital/odoo/commit/e40553723b893bcd2244e9b9fcc5c1a7e1bf27d3

Reply:
```
- MO/picking/PO/stock.move have now the "same" `priority` fields
(Boolean logic: Normal or Urgent).
```

Yeah that's not going to play nicely at all with how we want to use priorities in UDES...! Will raise an issue in the morning. :)

Reply:
Will resolve as not part of this story